datamade/how-to
# Initialize Heroku deployment documentation

## Overview

Initialize a first draft of documentation for deploying apps with Heroku.

I'm going to leave this open until we've successfully finished our deployment of https://github.com/datamade/mn-election-archive to make sure that we hit all the bases.

## Testing Instructions

* Read the rich Markdown preview and confirm it looks good
* Follow new Contents links and confirm they work
